frontend/libs/features/calculators/src/components/calculator/calculator.component.html

```diff
@@ -48,11 +48,11 @@ <h2 class="text-2xl font-bold">{{ title }}</h2>
 
   <div class="flex flex-col gap-4 md:flex-row md:items-start print:flex-row print:items-start">
     <!-- input block -->
-    <div class="bg-white rounded-2xl shadow w-full px-5 py-3">
+    <div class="bg-base-100 rounded-2xl shadow w-full px-5 py-3">
       <!-- calculation specific inputs -->
       <ng-content select="[form]" />
 
-      <div class="-mx-5 border mt-8 mb-3"><!--divider--></div>
+      <div class="divider"></div>
```

Comment:
De divider is niet meer de gehele width van het blok waar ie in zit. Mischien kun je dit fixen door de `-mx-5` terug te plaatsen?